test/semantics/blockconstruct03.f90

```diff
@@ -0,0 +1,60 @@
+! Copyright (c) 2019, ARM Ltd.  All rights reserved.
+!
+! Licensed under the Apache License, Version 2.0 (the "License");
+! you may not use this file except in compliance with the License.
+! You may obtain a copy of the License at
+!
+!     http://www.apache.org/licenses/LICENSE-2.0
+!
+! Unless required by applicable law or agreed to in writing, software
+! distributed under the License is distributed on an "AS IS" BASIS,
+! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+! See the License for the specific language governing permissions and
+! limitations under the License.
+! Tests implemented fro this standard
+! 11.1.4 - 4 It is permissible to branch to and en-block-stm only withinh its
+!            Block Construct
+! C1109
+! RUN: ${F18} -fdebug-semantics   %s 2>&1 | ${FileCheck} %s
+! CHECK: label '20' is not in scope
+! CHECK: should be
```

Comment:
Done

Reply:
I dont belive I can test all of them as ERROR, because:
  -> label %s is not in scope, and
 -> should be 
are not accused as an ERROR by the compiler, so I imagine it will not work as ERROR. Right? 
Maybe that is the reason the other tests only do CHECK.